### PR TITLE
fix: do not use Me as abbreviation for message extension

### PR DESCRIPTION
**Pull request by SLdragon**
https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/12664226

### CODE REVIEW

**Comment by SLdragon** [comment]
E2E TEST: Only wording change.

**Review by KennethBWSong — DISMISSED** [review]

File: templates/msgext/js/default/index.js [hunk 1]
```diff
@@ -40,7 +40,7 @@ const messageExtensionBot = new MessageExtensionBot();
 // Create HTTP server.
 const server = express();
 const port = process.env.port || process.env.PORT || 3978;
-server.listen(port, () => console.log(`\Bot/ME service listening at https://localhost:${port}`));
+server.listen(port, () => console.log(`\Bot/Message Extension service listening at https://localhost:${port}`));
```
Comment (IvanJobs) [inline templates/msgext/js/default/index.js:43]: Would `Messaging extension` be better? refer to official docs like https://docs.microsoft.com/en-us/microsoftteams/platform/messaging-extensions/what-are-messaging-extensions

Comment (SLdragon) [reply templates/msgext/js/default/index.js:43]: Ok, thanks, will update

**Review by IvanJobs — APPROVED** [review]
lgtm for bot

**Review by KennethBWSong — APPROVED** [review]

**Review by LongOddCode — APPROVED** [review]

**Review by 1openwindow — APPROVED** [review]

**Review by HuihuiWu-Microsoft — APPROVED** [review]

**Review by adashen — APPROVED** [review]

**Pull request merged**